tensorboard/plugins/projector/polymer3/vz_projector/vz-projector-app.ts

```diff
@@ -102,43 +103,19 @@
       >
       </vz-projector>
     </div>
-  </template>
-  <!-- Google analytics -->
```

Comment:
Is Google Analytics moved somewhere else, or removed intentionally?

Reply:
Added it back but only at the standalone app level.